api_server.py

```
@@ -364,19 +377,54 @@ async def root():
+        # ChromaDB (critical)
+        try:
+            await run_in_threadpool(vector_store.get_collection_stats)
+            checks["chromadb"] = "ok"
+        except Exception:
+            checks["chromadb"] = "error"
+            status = "unhealthy"
+
+        # Embeddings (non-critical — lazy singleton, None means not yet loaded)
+        try:
+            checks["embeddings"] = "ok"  # ponytail: not_loaded is normal lazy-init; only error on exception
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)